src/django/api/admin.py

```diff
@@ -296,6 +373,29 @@ class EmailAddressAdmin(admin.ModelAdmin):
     list_filter = ('verified', 'primary')
 
 
+class WageIndicatorCountryDataAdmin(admin.ModelAdmin):
+    list_display = ('country_code', 'living_wage_link_national',
+                    'minimum_wage_link_english', 'minimum_wage_link_national')
+    search_fields = ('country_code',)
+    fields = ('country_code', 'living_wage_link_national',
+              'minimum_wage_link_english', 'minimum_wage_link_national',
+              'created_at', 'updated_at')
+
+    def get_readonly_fields(self, request, obj=None):
+        '''
+        Make country_code readonly when editing, but editable when creating.
+        '''
+        if obj:  # Editing existing object.
```

Comment:
According to best practices the comments should answer the question `Why` something is being done a certain way, this seems like it's just describing `What` is being done.